src/tests/ftest/io/io_consistency.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/python
+"""
+  (C) Copyright 2020 Intel Corporation.
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+     http://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+  GOVERNMENT LICENSE RIGHTS-OPEN SOURCE SOFTWARE
+  The Government's rights to use, modify, reproduce, release, perform, display,
+  or disclose this software are subject to the terms of the Apache License as
+  provided in Contract No. B609815.
+  Any reproduction of computer software, computer software documentation, or
+  portions thereof marked with this legend must also reproduce the markings.
+"""
+
+from ior_test_base import IorTestBase
+
+
+class IoConsistency(IorTestBase):
+    """Test class Description: Verify data consistency using different
+                               middlewares. In this case, using DFS and POSIX.
+
+    :avocado: recursive
+    """
+
+    def tearDown(self):
+        """Tear down each test case."""
+        try:
+            if self.dfuse:
+                self.dfuse.stop()
+        finally:
+            # Stop the servers and agents
+            super(IoConsistency, self).tearDown()
+
+    def test_ioconsistency(self):
+        """Jira ID: DAOS-4778.
+        Test Description:
+            Run IOR first using DFS and then using POSIX to verify data
+            consistency.
+        Use Cases:
+            Create a pool
+            Create POSIX type container.
+            Run ior -a DFS with FPP and keep the file.
+            Mount the container using dfuse
+            Run IOR -a POSIX -r -R to verify data consistency and delete the
+            file.
+            Try to re-write to the same file name after deletion, which should
```

Comment:
Is this the same step as your comment in Line 73? If so, why one is re-write and the other is re-create?

Reply:
It's kind of a same thing. But I can change it to 're-create' at both places.